src/viking.js

```diff
@@ -1,11 +1,100 @@
 // Soldier
-class Soldier {}
+class Soldier {
+  constructor(health,strength){
+      this.health = health
+      this.strength = strength
+  }
+  attack() {
+      return this.strength
+  }
+  receiveDamage(damage) {
+      this.health -= damage
+  }
+
+}
 
 // Viking
-class Viking {}
+class Viking extends Soldier {
+    constructor(name,health,strength){
+       super(health,strength)
+       this.name = name
+
+    }
+    attack() {
+        return this.strength
+    }
+    receiveDamage(damage) {
```

Comment:
You could also reuse the receiveDamage function from Soldier using _super_: 
` receiveDamage (damage) {`
  `  super.receiveDamage(damage)`
   ` if (this.health > 0) {`
  `    return ${this.name} has received ${damage} points of damage`
 `   } else {`
 `     return ${this.name} has died in act of combat`
 `   }`
`  }`